blacklight-cornell/app/controllers/catalog_controller.rb

```diff
@@ -728,20 +728,12 @@ def afemail
     @response, @documents = search_service.fetch docs
     dox = {to: "jgr25@cornell.edu", message: "your stuff", callnumber:  @id}
     email_action(dox)
-    # , to: "jgr25@cornell.edu", message: "your stuff", :callnumber => docs
-
-    Rails.logger.info("es287_debug #{__FILE__}:#{__LINE__}  params = #{params.inspect}")
-  end
-
-  def logins
```

Comment:
This method seemed to have no purpose besides executing a logger method.

Reply:
There's also a commented out route in config/routes - seems like you can get rid of this controller action and commented out route. There are also a few reference to `logins` elsewhere in the code and feature specs - seems to be tied to an old google login feature? All of this seems like they can get removed, but this is substantial enough that might be better to do in a separate, dedicated PR. ...Perhaps it's better to do all the code cleanup in a separate PR actually?

Reply:
After looking into this more is seems to touch a few more files. I create a ticket for this for a seperate cleanup branch: https://culibrary.atlassian.net/browse/DACCESS-765